test/helpers/test.jl

```diff
@@ -41,6 +41,7 @@ end
 module FileCheck
     import LLVM_jll
     import IOCapture
+    using GPUCompiler, LLVM
     using InteractiveUtils
     using Test
```

Comment:
That means we don't need InteractiveUtils here anymore.

(I was half thinking about spinning this out as a small package, but I am unsure if that's worthwhile, but it might be nice to use this downstream in CUDA or Enzyme, but let's first see if we like it here.)